src/schema/index.ts

```diff
@@ -181,6 +184,7 @@ export const schema = new Schema<Nodes, Marks>({
     section,
     section_label: sectionLabel,
     section_title: sectionTitle,
+    section_title_plain: sectionTitlePlain,
```

Comment:
It's not clear why we need to create an alias to this not without any functional change?

Reply:
This is because there's a corresponding change in the body-editor, which allows us to use non-editable version of the section title element.

Reply:
Got it. Can we please either rename it to non_editable or add a comment here that it's for non-editable titles?

Reply:
Done, comment added.